decidim-core/spec/commands/decidim/update_account_spec.rb

```diff
@@ -17,7 +17,8 @@ module Decidim
         remove_avatar: nil,
         personal_url: "https://example.org",
         about: "This is a description of me",
-        locale: "es"
+        locale: "es",
+        time_zone: "UTC"
```

Comment:
```suggestion
        time_zone: timezone
```
As timezone is not strictly mandatory, we can define a `let` variable instead an test that it updates if defined and does nothing if not.
```ruby
let(:timezone) { "UTC" }
```